identity-apps-core/apps/accounts/src/main/webapp/includes/cookie-consent-banner.jsp

```diff
@@ -0,0 +1,58 @@
+<%--
+  ~ Copyright (c) 2021-2025, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Let's set the header year to 2025 for all files, as they were created in that year.

Reply:
Check all other pages as well